app/templates/components/donation/card-item.hbs

```diff
@@ -1 +1,5 @@
-{{card.brand}} ending in {{card.last4}}
\ No newline at end of file
+{{#if card.isLoading}}
+  Loading card...
+{{else}}
+  {{card.brand}} ending in {{card.last4}}
+{{/if}}
```

Comment:
A quick way to resolve the case where the record is still loading, so we just get `<blank> ending in <blank>`